<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metis Vela</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1171</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Connection</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>411</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Device Selection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>BaudRate Selection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Other Baudrate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="DeviceButton">
           <property name="text">
            <string>Check Device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="DeviceList"/>
         </item>
         <item>
          <widget class="QComboBox" name="BaudList"/>
         </item>
         <item>
          <widget class="QPushButton" name="ConnectionButton">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="persBaud"/>
         </item>
         <item>
          <widget class="QPushButton" name="addValueButton">
           <property name="text">
            <string>Add Personal Value</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="receivedText">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>400</y>
        <width>1071</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="connectionInfo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>411</width>
        <height>101</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lostPackLCD">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>362</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>370</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Lost Data Pack</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="connTimeLCD">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>322</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>330</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Connection Time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Telemetry</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>911</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="telPlot"/>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>0</y>
        <width>231</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Wind Data</string>
      </property>
      <widget class="QLCDNumber" name="windSpeed">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="numDigits">
        <number>4</number>
       </property>
      </widget>
      <widget class="QGraphicsView" name="windDirView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>270</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Attitude</string>
      </property>
      <widget class="QLCDNumber" name="pitchLCD">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="numDigits">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="yawLCD">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="numDigits">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Yaw</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="rollLCD">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="numDigits">
        <number>4</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Data Analysis</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Infusion Check</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>411</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="infusionPlot"/>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdTime">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="connStatus">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>70</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1266</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
